site/gulpfile.js

```diff
@@ -1,15 +1,18 @@
 var gulp = require('gulp');
 var elixir = require('laravel-elixir');
+var gutils = require('gulp-util');
 
 elixir.config.assetsPath = 'source/_assets';
 elixir.config.publicPath = 'source';
 
-elixir(function(mix) {
+var $environment = gutils.env.env || 'local';
```

Comment:
Read in `--env=<environment>` from the gulp command line and assign it. If it is not provided use `local` as the environment to build.